opencti-platform/opencti-graphql/src/domain/task-common.js

```diff
@@ -50,9 +51,8 @@ export const createDefaultTask = (user, input, taskType, taskExpectedNumber) =>
   };
 };
 
-export const createListTask = async (user, input) => {
+const buildListTask = async (user, input) => {
```

Comment:
You can keep the old method and add a check in checkActionValidity in entity type 
What do you think ?

Reply:
right! let's do this

Reply:
but we'll have to pass 'context' to createListTask event if it is not usefull (it's only useful in case of notifications)...